src/senselab/audio/tasks/speaker_verification/speaker_verification.py

```diff
@@ -0,0 +1,81 @@
+"""Audio Processing and Speaker Verification Module.
+
+This module provides functions for resampling audio using an IIR filter and
+verifying if two audio samples or files are from the same speaker using a
+specified model.
+"""
+
+import typing as ty
+
+import torch
+from scipy import signal
+from speechbrain.augment.time_domain import Resample
+from speechbrain.inference.speaker import SpeakerRecognition
+
+from senselab.audio.data_structures.audio import Audio
+
+
+def _resample_iir(audio: Audio, lowcut: float, new_sample_rate: int, order: int = 4) -> Audio:
```

Comment:
we do have resampling already in `senselab`. if you think some changes/improvements are necessary, please edit the existing method and use that instead of creating a new one

Reply:
Updated resample_audios.